docs/reference/aggregations/bucket/multi-terms-aggregation.asciidoc

```diff
@@ -0,0 +1,424 @@
+        }]
+      }
+    }
+  }
+}
+--------------------------------------------------
+// TEST[s/_search/_search\?filter_path=aggregations/]
+
+<1> `multi_terms` aggregation can work with the same field types as a
+<<search-aggregations-bucket-terms-aggregation-order,`terms aggregation`>> and supports most of the terms aggregation parameters.
+
+Response:
+
+[source,console-result]
+--------------------------------------------------
+{
+  ...
+  "aggregations" : {
+    "genres_and_products" : {
+      "doc_count_error_upper_bound" : 0,  <1>
+      "sum_other_doc_count" : 0,          <2>
+      "buckets" : [                       <3>
+        {
+          "key" : [
+            "rock",
+            "Product A"
+          ],
+          "key_as_string" : "rock|Product A",
+          "doc_count" : 2
+        },
+        {
+          "key" : [
+            "electronic",
+            "Product B"
+          ],
+          "key_as_string" : "electronic|Product B",
+          "doc_count" : 1
+        },
+        {
+          "key" : [
+            "jazz",
+            "Product B"
+          ],
+          "key_as_string" : "jazz|Product B",
+          "doc_count" : 1
+        },
+        {
+          "key" : [
+            "rock",
+            "Product B"
+          ],
+          "key_as_string" : "rock|Product B",
+          "doc_count" : 1
+        }
+      ]
+    }
+  }
+}
+--------------------------------------------------
+// TESTRESPONSE[s/\.\.\.//]
+
+<1> an upper bound of the error on the document counts for each term, see <<search-aggregations-bucket-multi-terms-aggregation-approximate-counts,below>
+<2> when there are lots of unique terms, Elasticsearch only returns the top terms; this number is the sum of the document counts for all buckets that are not part of the response
+<3> the list of the top buckets.
+
+By default, the `multi_terms` aggregation will return the buckets for the top ten terms ordered by the `doc_count`. One can
+change this default behaviour by setting the `size` parameter.
+
+[[search-aggregations-bucket-multi-terms-aggregation-parameters]]
+==== Aggregation Parameters
+
+The following parameters are supported. See <<search-aggregations-bucket-terms-aggregation-order,`terms aggregation`>> for more detailed
+explanation of these parameters.
+
+[horizontal]
+size::                        Optional. Defines how many term buckets should be returned out of the overall terms list. Defaults to 10.
+
+shard_size::                  Optional. The higher the requested `size` is, the more accurate the results will be, but also, the more
+                              expensive it will be to compute the final results. The default `shard_size` is `(size * 1.5 + 10)`.
+
+show_term_doc_count_error::   Optional. Calculates the doc count error on per term basis. Defaults to `false`
+
+order::                       Optional. Specifies the order of the buckets. Defaults to the number of documents per bucket. The bucket terms
```

Comment:
Hmm. It is there, but it looks like it is difficult to find. I will rename `Sub aggregation examples` into `Sub aggregation and sorting examples`, hopefully it will make it easier to spot.

Reply:
Ah gotcha :+1: